### PR TITLE
addendum regarding omero.web.check_version

### PR DESCRIPTION
**Pull request by kennethgillen**
Addendum re: `omero.web.check_version`

Renders like:

screen shot 2017-10-05 at 10 13 14

### CODE REVIEW

**Comment by hflynn** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/security/advisories/2017-SV4-guest-user/

**Comment by kennethgillen** [comment]
screen shot 2017-10-05 at 11 27 25

File: security/advisories/2017-SV4-guest-user/index.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h3 class="secvul-heading"><i class="fa fa-history"></i> Background</h3>
             <div class="small-12 medium-9 columns">
                 <p class="secvul-description">A "guest" user is enabled by default on all OMERO systems to allow clients to request non-sensitive information on startup e.g. the current server version. A few commands failed to check for the guest user and therefore allowed user-like actions. This included current sessions requests which allowed "bin/omero sessions who" to disclose information about when other users were logged in. Additionally,  "bin/omero script upload" would permit the upload of files.
                 </p>
-                <p class="secvul-description">A review was performed of all API methods, disabling all but the bare minimum of calls that are needed by "guest". Future versions may loosen the restriction to re-allow some minimal actions by "guest".
+                <p class="secvul-description">A review was performed of all API methods, disabling all but the bare minimum of calls that are needed by "guest". Future versions may loosen the restriction to re-allow some minimal actions by "guest". Note that this will also disable the password reset functionality in OMERO.web.
```
Comment (hflynn) [inline security/advisories/2017-SV4-guest-user/index.html]: Might be clear to put this after "The fix..." and say "This has also disabled the password reset functionality in OMERO.web"

Comment (mtbc) [reply security/advisories/2017-SV4-guest-user/index.html]: Admins *can* still reset others' passwords, I think it's just ordinary users can't use the "forgotten password" thing.

Comment (kennethgillen) [reply security/advisories/2017-SV4-guest-user/index.html]: @mtbc  - how does https://github.com/openmicroscopy/www.openmicroscopy.org/pull/144/commits/58032c377a4d8127724f580f8d92c922d97c6fc1 read?

Comment (mtbc) [reply security/advisories/2017-SV4-guest-user/index.html]: Certainly eliminates the ambiguity for me, thank you.

**Comment by hflynn** [comment]
https://snoopycrimecop.github.io/www.openmicroscopy.org/security/advisories/2017-SV4-guest-user/ up to date now, should be ready to merge and deploy after lunch

**Comment by kennethgillen** [comment]
Deployed on http://beluga.openmicroscopy.org:8000/reports/aacb56d9-a261-4df2-b032-32a7024c3bd2.html

**Comment by kennethgillen** [comment]
Deployed this afternoon.

**Pull request merged**